source/core/box.ts

```diff
@@ -0,0 +1,225 @@
+
+import { assert } from '../auxiliaries';
+
+import { Buffer } from '../buffer';
+import { Context } from '../context';
+import { Geometry } from '../geometry';
+
+
+/**
+ * Geometry of a box with configurable size and texture coordinates (optional).
+ */
+export class Box extends Geometry {
```

Comment:
please change this to `BoxGeometry` - all geometry inheritors should describe themselves somehow as geometry, e.g., in another project we have `CuboidGeometry`, `QuadGeometry`, and we have `LabelGeometry`, `NDCFillingRectangle`, and `NDCFillingTriangle`. The last two are not intended for typical geometry use, thus, the deviation from the naming scheme.